sematic/ui/src/components/TrackingNotice.tsx

```diff
@@ -0,0 +1,105 @@
+import { useAtom } from "jotai";
+import { atomWithStorage } from "jotai/utils";
+import { styled } from "@mui/system";
+import Link from "@mui/material/Link/Link"
+import Button from "@mui/material/Button/Button"
+import Dialog from "@mui/material/Dialog/Dialog"
+import DialogActions from "@mui/material/DialogActions/DialogActions"
+import { alertClasses, DialogContent, DialogContentText, DialogTitle, FormGroup, FormControlLabel, Checkbox } from "@mui/material";
+import React, { useState, useMemo, useCallback } from "react";
+import { spacing } from "../utils";
+import { applyPostHogOptOutSetting, optOutStorageKey } from "../postHogManager"
+
+const PageFooterContainer = styled('div', {
+  shouldForwardProp: () => true
+})`
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  width: fit-content;
+
+  & .${alertClasses.root}:hover {
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`;
+
+const StyledNoticeText = styled('div')`
+  padding: ${spacing(2)};
+  cursor: pointer;
+`;
```

Comment:
Can we make this `theme.palette.grey[100]` or something like that?

Reply:
This is `theme.palette.grey[100]` and barely visible.

<img width="283" alt="image" src="https://user-images.githubusercontent.com/1046489/219558607-bce68018-e84d-4d51-be0b-5449a30d61b6.png">


How about `[400]`? which looks like this:

<img width="272" alt="image" src="https://user-images.githubusercontent.com/1046489/219558499-d9ab5e52-3a05-48db-8f1b-6b9d21efedd7.png">